src/shared/ui/SearchForm/SearchForm.test.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { SearchForm } from "./SearchForm";
+import { keyboard } from "@testing-library/user-event/dist/keyboard";
+
+describe("SearchForm Component Functionality", () => {
+  test("Should Render Initial Value", () => {
+    const initialValue = "Test";
+    render(<SearchForm initialValue={initialValue} />);
+    const inputElement = screen.getByRole("textbox");
+    const inputValue = inputElement.value;
+    expect(inputValue).toBe(initialValue);
+  });
+  test("Should onChange fire with right value when clicked submit after typing", () => {
+    const mockOnChange = jest.fn((e) => e.preventDefault());
```

Comment:
when you mock a fun you don't need to copy its functionality 
```suggestion
    const mockOnChange = jest.fn();
```

Reply:
test throws an error without this line -  (e) => e.preventDefault()

Reply:
<img width="474" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/871353ce-72b2-44b2-b287-83a10d70b765">
<img width="366" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/e8cbaf3b-1a3f-411f-939b-1cd789529f92">

Reply:
<img width="433" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/3be93deb-e8d2-49fd-9955-4d8884269d19">
<img width="292" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/0de363e2-e2aa-4c33-81d1-ddbb874ef077">

Reply:
Interesting, I have an error whenever I delete this part
![Screenshot from 2023-09-15 12-22-55](https://github.com/AnnaMartis/Epam-project/assets/50263725/74af7e1a-59fc-4e3f-baf1-d8aff7379385)

Reply:
![Screenshot from 2023-09-15 12-22-31](https://github.com/AnnaMartis/Epam-project/assets/50263725/244675ea-3cc2-4f14-9912-40da269ebe44)

Reply:
maybe I should change some config?